packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1421,6 +1421,8 @@ class EditableText extends StatefulWidget {
   /// notification. It is possible that the event may occur outside of the
   /// immediate bounding box defined by the text field, although it will be
   /// within the bounding box of a [TextFieldTapRegion] member.
+  /// Called for each tap that occurs outside of the [TextFieldTapRegion] group
```

Comment:
This should be at the top of this doc snippet and replace the original comment:
`Called for each tap that occurs outside of the [TextFieldTapRegion] group when the text field is focused.`